jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -138,7 +138,7 @@ static void checkOffset(int offset, int length) {
 		 *			  an Object to compare
 		 * @param o2
 		 *			  an Object to compare
-		 * @return an int < 0 if object1 is less than object2, 0 if they are equal, and > 0 if object1 is greater
+		 * @return an int is negative 0 if object1 is less than object2, 0 if they are equal, and greater than 0 if object1 is greater
```

Comment:
"is negative" should be "less than". From "and greater than" remove "and"

Reply:
Thanks @pshipton , I thought I have replaced this with {@code} .. let me see why my changes not reflected.

Reply:
@pshipton Looks like you are seeing bit old code changes. Let me see why it is not refreshed with latest.